### PR TITLE
P0 card component

### PR DESCRIPTION
**Pull request by dhimanaishwarya**
added cards

- Cards with badges
- Cards with dismiss
- Cards with text overlay
- Text only cards
- Vertical/Horizontal cards
- Cards with shadow

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+.card-horizontal {
+  min-width: 30rem;
+  padding: 1rem;
+  max-height: 15rem;
+}
+
+.card-heading {
+  padding: 8px;
```
Comment (Ananyamadhu08) [inline components/card/card.css:35]: use rem instead of px.

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go 👍

**Review by Debarshi95 — APPROVED** [review]
Looks good. Liked that you used semantic tags.

File: components/card/card.css (hunk 1 above)
Comment (dhimanaishwarya) [reply components/card/card.css:35]: hey, I am using rem for anything above 10 px.
Thanks for the review @Ananyamadhu08

**Pull request merged**